Send nearby packets from world list instead of server list

LuminousLoreBloom opened the pull request.

aikar reviewed (changes requested):
the player can be null

LuminousLoreBloom committed:
Send nearby packets from world player list

    No need to look in all servers

    Also makes it so nearby packets are sent from the world and not
    dimension. Because you can have two worlds with the same dimension in
    the server and hearing things like attack sound packets from another
    world in same dim is not ideal.
